mage_integrations/mage_integrations/destinations/kafka/__init__.py

```diff
@@ -0,0 +1,66 @@
+import argparse
+import json
+import sys
+from typing import Dict, List
+
+import pandas as pd
+from kafka import KafkaProducer
+
+from mage_integrations.destinations.base import Destination
+from mage_integrations.destinations.constants import KEY_RECORD
+from mage_integrations.destinations.utils import update_record_with_internal_columns
+from mage_integrations.utils.dictionary import merge_dict
+
+MAXIMUM_BATCH_SIZE_MB = 1
+
+
+class Kafka(Destination):
+
+    def test_connection(self) -> None:
+        raise Exception('Not Implemented')
```

Comment:
Are you able to implement this method?

Reply:
I tried using `bootstrap_connected` method from kafka-python but wasn't able to get it working
Solved it by creating a KafkaConsumer connection and checking if topics exists